langDir/en.json

```
@@ -520,7 +520,8 @@
     "sold": "Sold",
     "holdings": "Holdings",
     "gains": "Gains",
-    "showQr": "Show QR"
+    "showQr": "Show QR",
+    "collectedFromCreator": "I have {0} NFT(s) from this creator"
```

How do you like this?
```suggestion
    "collectedFromCreator": "Collected {0} NFT(s) from this creator"
```

---

thanks for your idea